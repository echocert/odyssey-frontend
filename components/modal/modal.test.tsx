import { render } from "@testing-library/react";
import "@testing-library/jest-dom";
import { create } from "react-test-renderer";
import Modal from "./modal";
describe("Modal component", () => {
  it("renders the Modal ", () => {
    render(<Modal />);
  });

  it("Verify the snapshot of the Modal is matched", () => {
    const wrap = create(<Modal />).toJSON();
    expect(wrap).toMatchSnapshot();
  });
});