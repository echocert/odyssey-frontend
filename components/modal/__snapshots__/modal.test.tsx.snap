// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Modal component Verify the snapshot of the Modal is matched 1`] = `null`;
